### PR TITLE
NEW Base skeleton for Katka FE

### PR DESCRIPTION
**Pull request by CCardosoDev**

### CODE REVIEW

File: Dockerfile-test [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Extending image
+FROM node:carbon
+
+# Create app directory
+RUN mkdir -p /app
```
Comment (mattiassluis) [inline Dockerfile-test]: I doubt we need the -p here since `/` hopefully exists

File: Dockerfile-test [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+WORKDIR /app
+
+# Versions
+RUN npm -v
```
Comment (mattiassluis) [inline Dockerfile-test]: Could you explain the purpose of printing the versions?

File: package.json [hunk 3]
```diff
@@ -0,0 +1,31 @@
+{
+  "name": "katka",
+  "version": "0.1.0",
+  "private": true,
```
Comment (mattiassluis) [inline package.json]: Since it is on Github should this not be set to false?

File: public/index.html [hunk 4]
```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <meta name="theme-color" content="#000000">
+    <!--
```
Comment (mattiassluis) [inline public/index.html]: Is this documentation stripped from the final output or will these comments also be visible for the end-user? In that case we should look at a different way of putting this.

File: src/App.test.js [hunk 5]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+import App from './App';
+
+it('renders without crashing', () => {
+  const div = document.createElement('div');
+  ReactDOM.render( < App / > , div);
+  ReactDOM.unmountComponentAtNode(div);
+});
```
Comment (mattiassluis) [inline src/App.test.js]: See files below: there is a typo on the `assets` folder: it is now called `assents`

File: src/components/Pipelines/Pipelines.css [hunk 6]
```diff
@@ -0,0 +1,3 @@
+.ui.grid>.row {
```
Comment (mattiassluis) [inline src/components/Pipelines/Pipelines.css]: I see this code being repeated often, can we maybe put it in a shared stylesheet or something like that so when it needs to be changed we don't need to update it in multiple places?

File: src/App.test.js (hunk 5 above)
Comment (CCardosoDev) [reply src/App.test.js]: Good one, changed it!

File: .vscode/settings.json [hunk 7]
```diff
@@ -0,0 +1,2 @@
+{
```
[the remark this answers is not in the document]
Comment (CCardosoDev) [reply .vscode/settings.json]: Removed, missed it.

File: Dockerfile-test (hunk 1 above)
Comment (CCardosoDev) [reply Dockerfile-test]: Before I was using a different directory that would have to create 2 folders in the process, but I changed my mind to make it simple, but forgot to remove `-p`.

File: Dockerfile-test (hunk 2 above)
Comment (CCardosoDev) [reply Dockerfile-test]: I think at the time I was using `latest`, but like this there is no need for that, removed it.

File: package.json (hunk 3 above)
Comment (CCardosoDev) [reply package.json]: Agreed, made it public.

File: public/index.html (hunk 4 above)
Comment (CCardosoDev) [reply public/index.html]: This is visible on inspect mode, will remove them.

File: src/components/Pipelines/Pipelines.css (hunk 6 above)
Comment (CCardosoDev) [reply src/components/Pipelines/Pipelines.css]: There isn't even a need for this style anymore, removed from both files.

**Pull request merged**